Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM alpine:3.8
+FROM quay.io/deis/lightweight-docker-go:v0.7.0
```

Comment:
The name of the image is misleading. (Can't recall the genesis of the name now.) This is just Go bundled with some common Go-related tools. It's been used broadly across many projects and source is available at github.com/deis/lightweight-docker-go

Reply:
Also, you could use a different Go image here. I opted to use the same one that is used for _other_ tasks like linting and unit tests so that we can minimize the set of images we're dependent on and so we can also have a single image that we can point to as being "our development environment."

Reply:
>This is just Go bundled with some common Go-related tools.

Ok great 👍

>  I opted to use the same one that is used for other tasks like linting and unit tests so that we can minimize the set of images we're dependent on 

Makes sense 👍